atlassian_jwt_auth/contrib/django/utils.py

```diff
@@ -7,7 +7,7 @@ def parse_jwt(verifier, encoded_jwt):
     claims = verifier.verify_jwt(
         encoded_jwt,
         settings.ASAP_VALID_AUDIENCE,
-        leeway=60
+        leeway=getattr(settings, 'ASAP_VALID_LEEWAY', 0)
```

Comment:
IMHO 60 seconds is too long when that's the suggested token lifetime. Allowing users to specify their own value and defaulting to zero seems fairly sane to me (given the short token lifetime) do you have issues with time synchronisation between severs that would require a value higher than say `5` here?

cc @mark-adams / @jeremyshoemaker